docs/src/pages/plugins/persistQueryClient.md

```diff
@@ -48,31 +50,91 @@ You can also pass it `Infinity` to disable garbage collection behavior entirely.
 
 ## How does it work?
 
-As you use your application:
+- A check for window `undefined` is performed prior to saving/restoring/removing your data (avoids build errors).
 
-- When your query/mutation cache is updated, it will be dehydrated and stored by the persister you provided. **By default**, this action is throttled to happen at most every 1 second to save on potentially expensive writes to a persister, but can be customized as you see fit.
+### Storing
 
-When you reload/bootstrap your app:
+As you use your application:
 
-- Attempts to load a previously persisted dehydrated query/mutation cache from the persister
-- If a cache is found that is older than the `maxAge` (which by default is 24 hours), it will be discarded. This can be customized as you see fit.
+- When your query/mutation cache is updated, it will be [`dehydrated`](../reference/hydration#dehydrate) and stored by the persister you provided. The officially supported persisters throttle this action to happen at most every 1 second to save on potentially expensive writes, but can be customized as you see fit.
 
-## Cache Busting
+#### Cache Busting
 
 Sometimes you may make changes to your application or data that immediately invalidate any and all cached data. If and when this happens, you can pass a `buster` string option to `persistQueryClient`, and if the cache that is found does not also have that buster string, it will be discarded.
 
 ```ts
 persistQueryClient({ queryClient, persister, buster: buildHash })
 ```
 
+### Restoring
+
+When you reload/bootstrap your app:
+
+- Attempts to [`hydrate`](../reference/hydration#hydrate) a previously persisted dehydrated query/mutation cache from the persister back into the query cache.
+- If a cache is found that is older than the `maxAge` (which by default is 24 hours), it will be discarded. This can be customized as you see fit.
+
+### Removal
+
+- If data is found to be expired (see `maxAge`), busted (see `buster`), error (ex: `throws ...`), or empty (ex: `undefined`), the persister `removeClient()` is called and the cache is immediately discarded.
+
 ## API
 
+### `persistQueryClientRestore`
+
+This will attempt to restore a persister's stored cached to the active query cache.
+
+```ts
+persistQueryClientRestore({
+  queryClient,
+  persister,
+  maxAge = 1000 * 60 * 60 * 24, // 24 hours
+  buster = '',
+  hydrateOptions = undefined,
```

Comment:
just a question: what's the reason of defaulting to `undefined` ? If `hydrateOptions` isn't passed, it will be `undefined` anyhow 🤔

Reply:
When I was reading the docs before, the defaults and required parameters were a little difficult for me to follow.  My hope here was to indicate that the `maxAge`, `buster`, `hydrateOptions` etc. have defaults and aren't required.